dapps-staking pallet integration (#4)
AstarNetwork/swanky-node: README.md

Swanky node is the Substrate based blockchain configured to enable `pallet-contracts` and more features to help Wasm Smart Contract development.

## Features

> - pallet-contracts enabled

Reviewer: Can you specify which pallet-contract version will it targets? And if `unstable-feature` is enabled by default

> - Instant seal and Manual seal is used for Consensus. No aura and grandpa.

Reviewer: ```suggestion
- `grandpa` & `aura` consensus was removed. By default `instant-seal` is used - and `manual-seal` can also be used
```
